### PR TITLE
update the MAXFEDID according to changes in the FEDNumbering

### PR DESCRIPTION
**Pull request by deguio**

### CODE REVIEW

**Comment by deguio** [comment]
+1

File: DQM/L1TMonitor/interface/BxTiming.h [hunk 1]
```diff
@@ -28,6 +28,7 @@
 #include "DataFormats/FEDRawData/interface/FEDNumbering.h"
 #include "DQMServices/Core/interface/DQMEDAnalyzer.h"
 
+#define MAXFED 4097
```
Comment (fwyzard) [inline DQM/L1TMonitor/interface/BxTiming.h]: why redefine the value here ?
I'd rather just reuse FEDNumbering::MAXFEDID

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**